@here/olp-sdk-core/lib/utils/TileKey.ts

```diff
@@ -238,6 +238,30 @@ export class TileKey implements QuadKey {
         return Math.floor(mortonCode / 4);
     }
 
+    /**
+     * Checks if a quadkey is valid.
+     *
+     * The number of rows and columns must not be greater than the maximum number of rows and columns in the given level.
+     * The maximum number of rows and columns in a level is computed as 2 to the power of the level.
+     *
+     * @note As the JavaScript number type can hold 53 bits in its mantissa, only levels up to 26 can be
+     * represented in the number representation returned by [[mortonCodeFromQuadKey]].
+     * A level must be positive and can't be greater than 26.
+     *
+     * @param key The current quadkey.
+     * @return True if the quadkey is valid, false otherwise.
+     */
+    static isValid(key: QuadKey): boolean {
+        // tslint:disable-next-line:no-magic-numbers
+        if (key.level < 0 || key.level > 26) {
+            return false;
+        }
+        const dimensionAtLevel = Math.pow(2, key.level);
+        const rowValid = key.row >= 0 && key.row < dimensionAtLevel;
+        const columnValid = key.column >= 0 && key.column < dimensionAtLevel;
+        return rowValid && columnValid;
+    }
+
```

Comment:
Either add this to the commit message or split into a separate commit as this does not look like removal of deprecated code.

Reply:
will create separate MR